pkg/apis/work/v1alpha2/binding_types.go

```diff
@@ -385,6 +385,9 @@ const (
 	// FullyApplied represents the condition that the resource referencing by ResourceBinding or ClusterResourceBinding
 	// has been applied to all scheduled clusters.
 	FullyApplied string = "FullyApplied"
+
+	// Suspended represents the condition that the ResourceBinding or ClusterResourceBinding is suspended to schedule.
+	Suspended string = "Suspended"
```

Comment:
This is ambiguous as people can not tell what's suspended.

Reply:
As far as I know, the Rb resource is an abstract of scheduling, so maybe it's enough? And other conditions are `Scheduled string = "Scheduled"`, which are also a semantic related to scheduling, or could you give some advice?

Reply:
modified.